### PR TITLE
Add unit tests for classes in the admin/ directory

### PR DESCRIPTION
**Pull request by kienstra**
* So far, adds test classes for the `Admin\License` and `Admin\Upgrade` classes
* Adds more tests for the `Admin\Settings` class

### CODE REVIEW

File: php/admin/class-license.php [hunk 1]
```diff
@@ -41,7 +41,7 @@ public function init() {
 	 * Register any hooks that this component needs.
 	 */
 	public function register_hooks() {
-		add_filter( 'pre_update_option_block_lab_license_key', array( $this, 'save_license_key' ), 10, 1 );
+		add_filter( 'pre_update_option_block_lab_license_key', array( $this, 'save_license_key' ) );
```
Comment (kienstra) [inline php/admin/class-license.php:44]: Removed `10` and `1` as they're the defaults.

File: php/admin/class-import.php [hunk 2]
```diff
@@ -204,7 +204,7 @@ public function validate_upload( $file ) {
 	 *
 	 * @param array $blocks An array of Block Lab content blocks.
 	 */
-	public function import( $blocks ) {
+	public function import_blocks( $blocks ) {
```
Comment (kienstra) [inline php/admin/class-import.php:207]: There was a warning that this method of `import()` has the same name as the class, `Import`, and could be confused with a constructor.

**Comment by kienstra** [comment]
Here's the current test coverage of the `admin/` directory:

admin-directory-coverage

File: php/admin/class-import.php (hunk 2 above)
Comment (kienstra) [reply php/admin/class-import.php:207]: This will require regression testing of importing, to make sure it doesn't break anything.

**Comment by kienstra** [comment]
**Request For Review**

Hi @lukecarbis,
Hope your weekend's off to a great start.

Could you please review this? The repeater is my next priority.

Thanks!

**Comment by lukecarbis** [comment]
Can confirm importing still works. :)

**Review by lukecarbis — APPROVED** [review]
Looks great @kienstra. Not sure why the code coverage doesn't show green across the board for the admin directory though?

Happy to see the inclusion of the `fixtures`, we can use that in the future to store mock block json.

**Comment by kienstra** [comment]
Hi @lukecarbis,
Thanks a lot for reviewing this.

> Looks great @kienstra. Not sure why the code coverage doesn't show green across the board for the admin directory though?

Good question. I think I can increase the coverage a little, I'll work on that.

>Happy to see the inclusion of the fixtures, we can use that in the future to store mock block json.

Ah, good point. I hadn't thought of that.

**Comment by lukecarbis** [comment]
Didn't mean to ask for more tests! Just based on my review, I thought you'd covered off on everything.

**Pull request merged**